django/urls/resolvers.py

```diff
@@ -128,9 +128,6 @@ def get_ns_resolver(ns_pattern, resolver, converters):
 
 
 class LocaleRegexDescriptor:
```

Comment:
I have turned this into two descriptor classes, one for `RegexPattern` and one for `RoutePattern`. The differences after adjusting were just enough to not be worth parametrizing.